io/zenoh-links/zenoh-link-quic/src/unicast.rs

```diff
@@ -418,3 +434,37 @@ async fn accept_task(
     }
     Ok(())
 }
+
+fn get_cert_common_name(conn: quinn::Connection) -> ZResult<QuicAuthId> {
```

Comment:
I think this function should accept `&quinn::Connection` as parameter and to not take ownership of the connection.